packages/react/src/select/popup/SelectPopup.tsx

```diff
@@ -91,8 +91,9 @@ const SelectPopup = React.forwardRef(function SelectPopup(
       )}
       <FloatingFocusManager
         context={positioner.positionerContext}
-        modal={false}
+        modal={modal}
         disabled={!mounted}
+        outsideElementsInert
       >
```

Comment:
```suggestion
      <FloatingFocusManager
        context={positioner.positionerContext}
        modal={false}
        modal={modal}
        disabled={!mounted}
        outsideElementsInert
        visuallyHiddenDismiss={modal ? 'Dismiss popup' : undefined}
      >
```

Reply:
:+1: I'm adding it to the Menu as well. We could make the text configurable in the future if needed.